static/js/hotkey.js

```diff
@@ -593,7 +594,7 @@ exports.process_hotkey = function (e, hotkey) {
         case 'compose': // 'c': compose
             compose_actions.start('stream', {trigger: "compose_hotkey"});
             return true;
-        case 'compose_private_message':
+        case 'compose_private_message': // 'x': compose PM
```

Comment:
Not sure if we'll need comments noting the migration? Keep it for now, and remove it if another reviewer thinks it should be removed.